synapse/appservice/api.py

```diff
@@ -128,8 +125,7 @@ def query_alias(self, service, alias):
             logger.warning("query_alias to %s threw exception %s", uri, ex)
```

Comment:
Hmm, the:

```
            if e.code == 404:
                return False
```

bit here is also useless.

Reply:
Why is that useless?

I'm unsure if you're asking me to make a change.

Reply:
Sorry, I had noticed you removed the `response = None` line above as it wasn't necessary.

I'd say it's useless as if that `if` statement doesn't trigger, then the function will `return False` after the `try` statement anyways.

Though it may've been that returning `True` if `code != 404` was the intention here.